lua/caramba/orchestrator.lua

```
@@ -0,0 +1,278 @@
+		end
+	end
+	return modules
+end
+
+local function find_files_for_module(module_name)
+	local root = utils.get_project_root()
+	local results = {}
+	-- Try several patterns: exact basename match across common code extensions
+	local exts = { 'lua','py','js','ts','tsx','jsx','go','rs','java','c','cpp','h','hpp' }
+	for _, ext in ipairs(exts) do
+		local pattern = string.format("%s/**/*%s%s.%s", root, module_name:sub(1,1) == '/' and '' or '', module_name:gsub('[/\\]', '*'), ext)
```

Luacheck (1.2.0)

[warning] line is too long (134 > 120)

(W631)